src/pybind/mgr/dashboard/frontend/src/app/ceph/block/rbd-form/rbd-form.component.ts

```diff
@@ -134,35 +135,40 @@ export class RbdFormComponent extends CdForm implements OnInit {
         requires: null,
         allowEnable: false,
         allowDisable: true,
-        helperHtml: $localize`Feature can be disabled but can't be re-enabled later`
+        helperHtml: $localize`Feature can be disabled but can't be re-enabled later`,
+        helperText: $localize`Deep flatten support speeds up the process of deleting a clone by removing the dependency on the parent image.`
```

Comment:
I think this kind of texts can clutter the screen a lot, let's not make them longer than they need to

```suggestion
        helperText: $localize`Speeds up the process of deleting a clone by removing the dependency on the parent image.`
```

Reply:
Done